domains/alexjoseph.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "This is my first githib page",
+  "repo": "https://github.com/alec1563/alexjoseph.github.io",
```

Comment:
This repo won't work for github pages. It has to be `githubusername/githubusername.github.io`. 

You should read through the [github pages docs](https://guides.github.com/features/pages/)